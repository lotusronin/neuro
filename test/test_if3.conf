conf = {
'src' : 'test_if3.nro',
'expected' : 'test_if3.expected'
}
