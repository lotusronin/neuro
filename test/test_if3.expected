first if passed
second if passed
